detect-new-secrets.sh

```diff
@@ -43,11 +43,11 @@ EOF
 
 generate_command_to_update_secrets_baseline() {
     cat << EOF > "$command_to_update_baseline_file"
-cat << 'NEW_BASELINE' > "$BASELINE_FILE"
+cat << 'NEW_BASELINE' > '$NEW_BASELINE'
 $(jq 'setpath(["results"]; (.results | map_values(. | map_values(setpath(["is_secret"]; (.is_secret // false))))))' "$BASELINE_FILE")
 NEW_BASELINE
 
-git commit -m "Updating baseline file" "$BASELINE_FILE"
+git commit -m 'Updating baseline file' '$NEW_BASELINE'
```

Comment:
This will use `NEW_BASELINE` literally instead of interpolating it, is that fine?

Reply:
There's a meta level here, the operative thing is the preceding line:

```sh
   cat << EOF
...
EOF
```

And _it_ _is_ interpolating.

But the next layer, what the user runs won't want to do _further_ interpolating.